wurst/util/PlayerUtils.wurst

```diff
@@ -0,0 +1,104 @@
+package PlayerUtils
+import LinkedList
+
+/**
+*   Credits to TriggerHappy for making PlayerUtils [vJass]
+*    - Link : https://www.hiveworkshop.com/threads/playerutils.278559/
+**/
+
+User array USERS
+
+public function player.toUser() returns User
+    return USERS[this.getId()]
+
+public class User
```

Comment:
I don't really like the class named User if the package is named PlayerUtils

Reply:
how about changing the name to PlayerData ? unless you have a better suggestion for a name?